apps/src/codebridge/FileTabs/index.tsx

```diff
@@ -57,12 +57,17 @@ export const FileTabs = React.memo(() => {
     }
   }
   function handleDragStart(event: DragStartEvent) {
-    setDraggingFileId(event.active.id as string);
-    // set file to active
+    // Handle drag start only if the file is in the list of open files.
+    // This can get called when the close button is clicked, and we want to ignore
+    // it in this case.
+    if (source.files[event.active.id as string].open) {
+      setDraggingFileId(event.active.id as string);
+      setActiveFile(event.active.id as string);
+    }
```

Comment:
Noting for a follow-up that if a tab is selected, a user can keyboard-navigate to next tab, but then when they press enter, the focus is still on currently selected tab. User needs to press enter again to be able to select a different tab.

https://github.com/user-attachments/assets/fe7ec7f0-1725-4330-9de6-cc893a851c44

Reply:
discussed offline, I removed the tab role so one less tab is required when navigating around. The drag and drop library requires clicking enter to "end" your drag and drop action, which may be confusing and worth considering if we should modify.

Reply:
Thanks! And curious to see what @hannahbergam thinks about this topic.